components/src/jsMain/kotlin/dev/fritz2/components/alert.kt

```diff
@@ -0,0 +1,280 @@
+package dev.fritz2.components
+
+import dev.fritz2.binding.RootStore
+import dev.fritz2.binding.SimpleHandler
+import dev.fritz2.components.validation.ComponentValidationMessage
+import dev.fritz2.components.validation.Severity
+import dev.fritz2.dom.html.RenderContext
+import dev.fritz2.styling.StyleClass
+import dev.fritz2.styling.params.BasicParams
+import dev.fritz2.styling.params.styled
+import dev.fritz2.styling.theme.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+
+/**
+ * A component to display an alert consisting of an icon, title and description.
+ * Different styles based on severities are supported, as well as a number of different layout options.
+ *
+ * Currently the following severities are available:
+ * - Info
+ * - Success
+ * - Warning
+ * - Error
+ * Specifying a severity will change the alert's color scheme based on the colors defined in the application theme.
+ * If no severity is specified, 'info' will be used by default.
+ *
+ * Additionally, a number of different layout options are available. These are:
+ * - 'subtle': A subtle style using different shades of the severity's base color defined in the application theme.
+ * - 'solid': A solid style using the severity's color from the application theme and a solid white color for the icon,
+ * text and decorations.
+ * - 'Top-Accent': A variation of the 'subtle' variant with a decoration element at the top.
+ * - 'Left-Accent': A variation of the 'subtle' variant with a decoration element on the left.
+ * If no variant is specified, 'solid' is used by default.
+ *
```

Comment:
1) From the explanation in the kitchensink demo and from what I gather from these comments, I the "subtle" style is supposed to be default. However, if no variant is specified, the "solid" style appears. I assume that the demo text is correct and the default is incorrectly set in the component.

2)

Reply:
No, afaik the opposite is true: The comment and the demo should be changed though.
But Jens is the source of truth for this :-)

Reply:
The leftAccented and topAccented variants are all colored "subtle".

Reply:
You are right, Jens had already changed that, so the demo was off because he didn't change it there. Sorry!